src/components/results/sensitivity-analysis/paged-sensitivity-analysis-result.js

```diff
@@ -146,8 +152,13 @@ const PagedSensitivityAnalysisResult = ({
     ]);
 
     useEffect(() => {
-        fetchResult();
-    }, [fetchResult]);
+        if (sensiStatus === RunningStatus.RUNNING) {
```

Comment:
Don't we need to handle FAILED status?

Reply:
We don't need to check this status because we have to go through the RUNNING status anyway, and we won't get any results when we are in FAILED